recipes/wheel-inspect/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "wheel-inspect" %}
+{% set version = "1.7.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/wheel-inspect-{{ version }}.tar.gz
+  sha256: e3a930c841e1d6d233fd2d213ee6ce07e990006a32d73b0f89937b539db26af6
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  entry_points:
+    - wheel2json = wheel_inspect.__main__:main
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - attrs >=18.1
+    - entry-points-txt >=0.1.0
+    - headerparser >=0.4.0
+    - packaging >=17.1
+    - python >=3.6
+    - readme_renderer >=24.0
+    - wheel-filename >=1.1
+
+test:
+  imports:
+    - wheel_inspect
```

Comment:
Please test your wheel2json entry point in the test section. You can just do `wheel2json --help` or something.

Reply:
Unfortunately as mentioned in the description, `wheel2json` doesn't have any `--help`.
It can only take a wheel file as input.

Reply:
There is [test data](https://github.com/jwodder/wheel-inspect/tree/master/test/data) included in the source tarball. Please add as single file from the test data and use that to test that the console script is working properly.

Reply:
I didn't notice test data were in the sdist. Nice. I added a test. Thanks!